src/main/lrsql/init/oidc.clj

```diff
@@ -84,6 +115,21 @@
          :oidc-config (select-config config)}
         ex)))))
 
+(s/fdef admin-interceptors
+  :args (s/cat :config partial-config-spec)
+  :ret (s/every i/interceptor?))
+
+(defn admin-interceptors
+  "Given a webserver config, return a (possibly empty) vector of interceptors
+  for use with the admin API. These validate token claims and ensure an admin
+  account is made."
+  [{:keys [oidc-issuer] :as config}]
```

Comment:
This `config` var results in an unused binding warning.

Reply:
https://github.com/yetanalytics/lrsql/pull/207/commits/d2db9b892d0a5df1e39c6d2a7609f76d52f79493